triggers/daml/Daml/Trigger.daml

```diff
@@ -28,13 +28,21 @@ module Daml.Trigger
  , RegisteredTemplates(..)
  , registeredTemplate
  , RelTime(..)
+ , toACS
```

Comment:
Shouldn't we export the `ACSBuilder` _type_ as well?

Reply:
Indeed, done.